colinMarkDown.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "Tropical Storm Colin Code"
+output: html_document
+---
+
+## Data retrieval function
+Not sure if we want to include all this or not, but hooray it *works*!
+
+
+```{r global_options, warning=FALSE, message=FALSE}
+library(dataRetrieval)
```

Comment:
This is a great start, but I think the idea here is to source these functions (retryNWIS and getDischarge) from their script files so that our markdown file looks cleaner. You should be able to replace L11-87 w/ `source('getDischarge.R')`